src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-containerized-function-monitoring/get-started.mdx

```diff
@@ -0,0 +1,412 @@
+---
+title: "Get started with Lambda containerized functions"
```

Comment:
In my opinion 'Containerized lambda functions' good instead of 'Lambda containerized functions' in all places